app/javascript/dashboard/components/widgets/conversation/ConversationHeader.vue

```diff
@@ -142,10 +142,21 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-.text-truncate {
-  white-space: nowrap;
-  overflow: hidden;
-  text-overflow: ellipsis;
+.user {
+  min-width: 0;
+  margin-right: var(--space-normal);
+
+  .user--profile__meta {
+    min-width: 0;
+  }
+
+  .text-truncate {
```

Comment:
This text-truncate class here is already defined as a utility class, we remove the repeating styles.
```suggestion
  .user--name {
    width: 100%;
  }
```